releasetasks/templates/desktop/common_extras.yml.tmpl

```diff
@@ -15,6 +15,7 @@ build_props:
     mozharness_changeset: "{{ mozharness_changeset }}"
     build_number: {{ buildNumber }}
     partials: {% set pipe = joiner(",") %}{% for p, e in sorted(partial_updates.items()) %}{{ pipe() }}{{ "{}build{}".format(p, e["buildNumber"]) }}{% endfor %}
+    release_eta: {% if release_eta %}"{{ release_eta }}"{% else %}null{% endif %}
```

Comment:
If I understand correctly, this will be set for all buildbot builders. Is that wanted? It probably doesn't hurt, just wondering.